app.js

```diff
@@ -1,18 +1,20 @@
 const express = require('express');
-const cors = require('cors');
+const cookieParser = require('cookie-parser');
```

Comment:
if we're not using cookie parser right now I would suggest removing it as it will likely not be used if we're making use of aws cognito